build-logic/code-quality/src/main/kotlin/testng.checkerframework.gradle.kts

```diff
@@ -0,0 +1,27 @@
+import org.checkerframework.gradle.plugin.CheckerFrameworkExtension
+
+plugins {
+    id("org.checkerframework")
+}
+
+dependencies {
+    checkerFramework("org.checkerframework:checker:3.36.0")
+}
+
+plugins.withId("org.checkerframework") {
+    configure<CheckerFrameworkExtension> {
```

Comment:
`plugins { id("org.checkerframework") }` adds `checkerframework` plugin, so `plugins.withId(..)` is not needed here.

Something like `checkerframework {...}` should do.